src/rest.cpp

```diff
@@ -570,6 +571,55 @@ static bool rest_getutxos(HTTPRequest* req, const std::string& strURIPart)
     }
 }
 
+// A bit of a hack - dependency on a function defined in rpc/mining.cpp
+UniValue estimatesmartfee(const JSONRPCRequest& request);
```

Comment:
I think it's not worth passing through the RPC call `estimatesmartfee`.
Better directly use `feeEstimator.estimateFee(nBlocks);`.